dpdata/deepmd/comp.py

```diff
@@ -110,5 +110,7 @@ def dump(folder,
     except OSError:
         pass
     if data.get("nopbc", False):
-        os.mknod(os.path.join(folder, "nopbc"))
+        #os.mknod(os.path.join(folder, "nopbc"))
+        with open(os.path.join(folder, "nopbc"), "w") as fw_nopbc:
+            fw_nopbc.write("")
```

Comment:
I think we don't need this line. Only a `pass` is enough.

Reply:
Does it mean that "nopbc" is not supported in dump function ?

Reply:
When you call `open` with `w`, the file has already been created. So you don't need to write it.